docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -0,0 +1,310 @@
+// Node.js, npx, typescript, etc.?
+
+As explained in the <<synthetics-quickstart,quickstart>>, there are two ways to run synthetic tests:
+
+* Run as an inline journey -- copy/paste into `heartbeat.yml`
+* Run a test suite -- import your tests into Heartbeat
+
+Which option is right for you? That depends.
+If you only have a few tests, an inline journey might be the easiest workflow --
+you only need paste your tests into your `heartbeat.yml` file.
+if you have a complex test suite, or your tests need to live with your application code,
+you're likely better off setting up a synthetic test suite.
+
+[discrete]
+[[synthetics-inline-journey]]
+=== Run an inline journey
+
+The easiest way to run a synthetic test is by creating an inline journey.
+The `journey` keyword isn't required, and access to variables like `page` and `params` is automatic.
+
+Copy and paste your test steps into `heartbeat.yml`.
+Heartbeat will fork Node.js, schedule your tests, and start headless chrome.
+You don't need to worry about anything else.
+
+An example, `short.js`, is provided in the
+https://github.com/elastic/synthetics/tree/master/examples/inline[elastic/synthetics] GitHub repository:
+
+[source,js]
+----
+// test-homepage-hover.js
+step("load homepage", async () => {
+    await page.goto('https://www.elastic.co');
+});
+step("hover over products menu", async () => {
+    await page.hover('css=[data-nav-item=products]');
+});
+----
+
+To run this, or any other inline example locally, change into the directory of your test,
+and pipe the file contents to the `elastic-synthetics` command.
+
+For example:
+
+[source,sh]
+----
+cat examples/inline/short.js | npx synthetics --inline
+----
+
+If everything works as expected, you'll get the following response:
+
+[source,sh]
+----
+Journey: inline
+   ✓  Step: 'load homepage' succeeded (1831 ms)
+   ✓  Step: 'hover over products menu' succeeded (97 ms)
+
+ 2 passed (2511 ms)
+----
+
+The script can then be copied into your in your `heartbeat.yml`:
+
+[source,yml]
+----
+heartbeat.monitors:
+- type: browser
+  id: my-monitor
+  name: My Monitor
+  schedule: "@every 1m"
+  script: |-
+    step("load homepage", async () => {
+        await page.goto('https://www.elastic.co');
+    });
+    step("hover over products menu", async () => {
+        await page.hover('css=[data-nav-item=products]');
+    });
+----
+
+That's it! You can either spin up Heartbeat yourself, or jump to <<synthetics-quickstart-step-three>>
+of the Quickstart to use the provided Docker project template.
+
+[discrete]
+[[synthetics-test-suite]]
+=== Run a test suite
+
+If you have a suite of tests you'd like to implement, you can use Elastic synthetics as a library.
+In this method, you use Docker to run both Heartbeat and `elastic-synthetics`.
+
+// [discrete]
+// [[synthetics-suite-install]]
+// ==== Step 1: Install `elastic-synthetics`
+
+If you haven't already, clone the https://github.com/elastic/synthetics[elastic/synthetics] repository
```

Comment:
We published the package, so this shouldnt be necessary. Users can do `npm install -g @elastic/synthetics`